pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -27,7 +27,7 @@
     math,
     QubitDevice,
     BasisState,
-    QubitStateVector,
+    StatePrep,
```

Comment:
This should be added, not replaced (see [this comment](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/79#discussion_r1301585672)).

Reply:
Done